.gitignore

```diff
@@ -269,4 +296,5 @@ gradle-app.setting
 
 # # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
 # gradle/wrapper/gradle-wrapper.properties
-scenicView.properties
+
+# End of https://www.gitignore.io/api/gradle,java,jabref,intellij,eclipse,netbeans,windows,linux,macos,snapcraft
```

Comment:
`.snapcraft/travis_snapcraft.cfg` is the travis-encrypted file with my personal credentials enabling an upload to the Ubuntu app store.